src/main/webapp/js/user-page-loader.js

```diff
@@ -39,6 +39,9 @@ function showMessageFormIfLoggedIn() {
       })
       .then((loginStatus) => {
         if (loginStatus.isLoggedIn) {
+            if (loginStatus.username == parameterUsername){
+                document.getElementById('about-me-form').classList.remove('hidden');
```

Comment:
Can you break this line similar to the "message-form" element below to improve readability?

```
const aboutMeForm = document.getElementById('about-me-form');
aboutMeForm.classList.remove('hidden');
```